Jenkinsfile

```diff
@@ -175,6 +175,24 @@ try {
         }
       }
     },
+    'CPU: Clang 3.9 MKLDNN': {
+      node('mxnetlinux-cpu') {
+        ws('workspace/build-cpu-clang39') {
```

Comment:
I think we're trying to give each build task its own unique workspace.  I'd recommend we change this to:

```
ws('workspace/build-cpu-mkldnn-clang39')
```